src/Makevars

```diff
@@ -24,4 +24,4 @@ tmblib:
 	--no-save --no-restore compile.R '$(TMB_FLAGS)')
 
 clean:
-	rm -rf *.so *.o TMB/*.so TMB/*.o
+	rm -rf *.so *.o TMB/*.so TMB/*.o
```

Comment:
Please always leave a blank line at the end of every file.  (Otherwise you will eventually get a weird conversion issue for plain-text files between Linux and Windows.  This is why GitHub flags missing EOLs with a red circle and bar through it.)